### PR TITLE
[DI] Fix support for immutable setters in CallTrait

### PR DESCRIPTION
**Pull request by Lctrs**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no
| Tickets       | 
| License       | MIT
| Doc PR        | N/A

### CODE REVIEW

**Review by nicolas-grekas — APPROVED** [review]
For 4.4 as a bug fix to me.

**Comment by Lctrs** [comment]
Not in `4.3` ? This param has been introduced on this branch.

**Comment by nicolas-grekas** [comment]
4.3 then, you're right!

**Comment by Lctrs** [comment]
@nicolas-grekas Done !

File: src/Symfony/Component/DependencyInjection/Loader/Configurator/Traits/CallTrait.php [hunk 1]
```diff
@@ -18,16 +18,17 @@ trait CallTrait
...
+     * @param string $method       The method name to call
+     * @param array  $arguments    An array of arguments to pass to the method call
+     * @param bool   $returnsClone Whether the call returns the service instance or not
      *
      * @return $this
      *
      * @throws InvalidArgumentException on empty $method param
      */
-    final public function call($method, array $arguments = [])
+    final public function call($method, array $arguments = []/*, bool $returnsClone = false*/)
```
Comment (nicolas-grekas) [inline src/Symfony/Component/DependencyInjection/Loader/Configurator/Traits/CallTrait.php]: No need for the compat layer: the method is final, we *can* add the argument.

Comment (Lctrs) [reply src/Symfony/Component/DependencyInjection/Loader/Configurator/Traits/CallTrait.php]: Done.

**Comment by nicolas-grekas** [comment]
Thank you @Lctrs.

**Pull request merged**